config/custom-environment-variables.yaml

```diff
@@ -80,6 +80,7 @@ datastore:
     retry:
       __name: DATASTORE_SEQUELIZE_RETRY
       __format: json
+    captureBuildMetrics: DATASTORE_SEQUELIZE_ADDHOOK
```

Comment:
its actually a hook, in this PR - https://git.ouroath.com/cocktails-screwdriver/ouroath-kubernetes/pull/183 - @jithin1987 commented to use CAPTURE_BUILD_METRICS

maybe DATASTORE_SEQUELIZE_CAPTURE_BUILD_METRICS ?

can remove DATASTORE_SEQUELIZE, as its not related to DB

Reply:
If it's not under `sequelize:`, can remove `DATASTORE_SEQUELIZE` prefix and used something like `buildMetricsEnabled: CAPTURE_BUILD_METRICS_ENABLED`. 

Otherwise, I think you need to keep it. Something like `buildMetricsEnabled: DATASTORE_SEQUELIZE_CAPTURE_METRICS_ENABLED`.